src/Stratis.Bitcoin.Features.Consensus/ConsensusLoop.cs

```diff
@@ -405,15 +416,33 @@ public void ValidateBlock(ContextInformation context)
                 // Calculate the consensus flags and check they are valid.
                 context.Flags = this.NodeDeployments.GetFlags(context.BlockValidationContext.ChainedBlock);
 
-                int lastCheckpointHeight = this.checkpoints.GetLastCheckpointHeight();
-                if (context.BlockValidationContext.ChainedBlock.Height > lastCheckpointHeight)
+                // Check whether to use checkpoint to skip block validation.
+                context.BlockValidationContext.SkipValidation = false; 
+                if (this.nodeSettings.UseCheckpoints)
+                {
+                    int lastCheckpointHeight = this.checkpoints.GetLastCheckpointHeight();
+                    context.BlockValidationContext.SkipValidation = context.BlockValidationContext.ChainedBlock.Height <= this.checkpoints.GetLastCheckpointHeight();
```

Comment:
this should be 
` context.BlockValidationContext.SkipValidation = context.BlockValidationContext.ChainedBlock.Height <= lastCheckpointHeight;`

Reply:
Yes - I will fix that up.